### DISCUSSION TITLE
[Feature] Add Google Cast support

### DISCUSSION THREAD

**Discussion by ytzelf** [immich-app/immich#2575] [Feature Request]
### The feature

Hi,

It'd be great to be able to cast albums to an Android device, most of the mobile image viewers have that option.

Thanks a lot!

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #1614

**Reply by aisbergde** [reply]
@bo0tzz 
Chromecast support could be a bit different from Android TV (maybe, I am wrong). 
if it is different, maybe, the feature request could be reopened?

**Comment by aisbergde** [comment]
could someone please reopen this request?
Chromecast has nothing to to with Android TV and the current Android TV implementation will not solve this request.

**Reply by osirase** [reply]
I went to share some photos via my self-hosted immich instance at a friends to their TV but had to resort to using miracast which made the photos take up like 30% of the screen, very small. Would have been so awesome if Google Cast was supported, hoping this feature request gets reopened. like @aisbergde said, it's not related to Android TV and I believe this was closed in error, @bo0tzz

**Reply by aisbergde** [reply]
it looks like @bo0tzz or some one of the other developers is not willing to reopen.

Because of the missing Chrome Cast support and because of this not reopened issue I am still using Google Photos with my family. We want to show photos and other assets on a TV, and this works fine with Chromecast.

**Reply by osirase** [reply]
It's more likely they're prioritising other tasks and features, which is fine. I'm sure it will get attention eventually :)

**Reply by aisbergde** [reply]
#7030 - it looks like there is some progress
